pom.xml

```diff
@@ -432,6 +432,33 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>flatten-maven-plugin</artifactId>
+                    <version>1.2.2</version>
+                    <configuration>
+                        <outputDirectory>${project.basedir}/generated/src/main/resources/</outputDirectory>
+                        <updatePomFile>false</updatePomFile>
+                    </configuration>
+                    <executions>
+                        <!-- enable flattening -->
```

Comment:
It seems like this is the key change - adding the flattening step here leads to all of the new .flattened-pom.xml files being generated.

Reply:
Correct, whenever you enter the `install` phase, new .flattened-pom.xml files will be generated (if you didn't change any dependencies, there won't be anything new things to commit), also since this is in `<pluginManagement>` we have to add the plugin to the child poms for it to actually be called.